Kentico/xperience-by-kentico-sitefinity-migration-tool
# Feature/content import

# Motivation

Which issue does this fix? Fixes #5

## Checklist


## How to test

Update app settings based on documentation, run the console app using the StartImportContent method and check your XbyK site. Channels, Languages, Users, Content Types, Media Libraries and files, Content items and Pages should be imported.
